forte/sparse_ci/bitwise_operations.hpp

```diff
@@ -55,7 +55,13 @@ inline double ui64_bit_parity(uint64_t x) { return 1 - 2 * ((std::popcount(x) &
 /// @brief Bit-scan to find next set bit
 /// @param x the uint64_t integer to test
 /// @return the index of the least significant 1-bit of x, or if x is zero, returns ~0
-inline uint64_t ui64_find_lowest_one_bit(uint64_t x) { return std::countr_zero(x); }
+inline uint64_t ui64_find_lowest_one_bit(uint64_t x) { 
+    if (auto y = std::countr_zero(x); y == 64) {
+        return ~0;
+    } else {
+        return y;
+    } 
```

Comment:
This can be done in one line in this way:
```suggestion
    return (x == 0) ? ~uint64_t(0) : std::countr_zero(x);
```